spec/factories/whitehall_export/edition_factory.rb

```diff
@@ -52,5 +52,28 @@
         ]
       end
     end
+
+    trait :published do
+      transient do
+        previous_state { "submitted" }
+      end
```

Comment:
Do we make use of this transient attribute? I have a feeling it's a c+p from scheduled where we care more about it

Reply:
You caught me! I'll get rid of it.